### ISSUE TITLE
Link to video about iOS constraints leading to no where

### ISSUE THREAD

**Issue by beerline** [fluttercommunity/flutter_workmanager#569]
Hello! 
This link https://developer.apple.com/videos/play/wwdc2020/10063
in documentation is broken

**Comment by gruvw** [comment]
Yes, if someone could find a link to that video, it would be great. Is it that the video is not accessible from this link anymore, or the video has been removed all together ?

**Comment by kamilkomnacki** [comment]
I know it's probably not the same video, but it seems to be helpful with introduce to background processing on iOS:
 https://developer.apple.com/videos/play/wwdc2019/707/

It's called `Advances in App Background Execution`

**Comment by stwarwas** [comment]
There is an article of the same author:

https://uynguyen.github.io/2020/09/26/Best-practice-iOS-background-processing-Background-App-Refresh-Task/

Here is a discussion on Reddit about the video:

https://www.reddit.com/r/iOSProgramming/comments/1c5pddy/wwdc_video_deleted_background_execution/

**Comment by ened** [comment]
I've checked the WWDC video links in the documentation and they appear to be working correctly. The link https://developer.apple.com/videos/play/wwdc2019/707 leads to a valid video about 'Advances in App Background Execution'. If you're referring to a different broken link, please provide the specific link and I'll be happy to fix it.
